fetch-github-token-js/action.yml

```
@@ -0,0 +1,21 @@
+name: Get Ephemeral GitHub Token from CI Vault
+description: 'Fetch an ephemeral GitHub token from Vault using OIDC authentication'
+inputs:
+  vault-instance:
+    description: 'Vault instance to connect to (ci-prod or ci-dev)'
+    required: true
+  vault-role:
+    description: 'Vault role to assume for GitHub token retrieval. If not provided, it will be generated based on the workflow ref of the GH Action.'
+    required: false
+    default: ''
+  skip-token-revoke:
+    description: 'If true, skip revoking the GitHub token on exit'
+    required: false
+    default: 'true'
```

should be revoked by default:

- https://github.com/actions/create-github-app-token/blob/b96fde71c0080358ed6e2d162f11c612c92a97d1/action.yml#L20-L23

```suggestion
    default: 'false'
```